setup.py

```diff
@@ -60,7 +60,9 @@
     include_package_data=True,
     install_requires=[
         "importlib-metadata ; python_version<'3.8'",
-        "eth-ape>=0.1.0a20,<0.2.0",
+        "eth-ape>=0.1.0b1",
```

Comment:
```suggestion
        "eth-ape>=0.1.0b2",
```

Any reason not to do b2?